tests/integration/test_wpk/test_wpk_manager.py

```diff
@@ -35,6 +37,19 @@
 global valid_sha1_list
 valid_sha1_list = {}
 
+if not global_parameters.wpk_version:
+    raise Exception("The WPK package version must be defined by parameter. See README.md")
```

Comment:
This should be a `ValueError` since the parameter is not defined. Also, I would check `wpk_version` value as follows:
```python
if global_parameters.wpk_version is None:
    ... do stuff
```

Reply:
Fixed